### DISCUSSION TITLE
Add ability to bind default brightness keys on keyboard

### DISCUSSION THREAD

**Discussion by Swift8641** [xanderfrangos/twinkle-tray#583] [Ideas]
I have a Keychron K3 keyboard and it has dedicated brightness keys on it. Right now twinkle tray does not detect them when I try to set up key binding however the default windows brightness slider does. Is it possible to add the ability for twinkle tray to detect dedicated brightness keys on keyboards?

**Comment by Alexnoj** [comment]
+1 for detect brightness keys
